scripts/debian/publish.sh

```diff
@@ -44,7 +44,6 @@ S3_REGION_ARG="--s3-region=us-west-2"
 #>> Attempting to obtain a lock
 #/var/lib/gems/2.3.0/gems/deb-s3-0.10.0/lib/deb/s3/lock.rb:24:in `throw': uncaught throw #"Unable to obtain a lock after 60, giving up."
 DEBS3_UPLOAD="deb-s3 upload $BUCKET_ARG $S3_REGION_ARG \
-  --fail-if-exists \
```

Comment:
This line is here for a reason, and nothing in your PR justifies deleting it. Revert, or justify the change.

Reply:
Uh oh. Good catch! I forgot to remove it after I finished testing